From ec02c52ae5b968f6187e4cd9fa7697b9aede1c4d Mon Sep 17 00:00:00 2001
From: Abhay Krishna Arunachalam <arnchlm@amazon.com>
Date: Thu, 19 Sep 2024 15:36:00 -0700
Subject: [PATCH 10/13] Pin Packer Goss plugin version to 3.1.4

Upstream image-builder moved from installing packer-provisioner-goss through a script to installing it
via Packer config (https://github.com/kubernetes-sigs/image-builder/commit/c0b70ae37c4aac14c156fc7b907a9b35147757df).
In the Packer config, they just specify the major version but we're using the full semver to pin it to the older
version.

Signed-off-by: Abhay Krishna Arunachalam <arnchlm@amazon.com>
---
 images/capi/packer/config.pkr.hcl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/images/capi/packer/config.pkr.hcl b/images/capi/packer/config.pkr.hcl
index 476179944..28e76f2a7 100644
--- a/images/capi/packer/config.pkr.hcl
+++ b/images/capi/packer/config.pkr.hcl
@@ -5,7 +5,7 @@ packer {
       source  = "github.com/hashicorp/ansible"
     }
     goss = {
-      version = "~> 3"
+      version = ">= 3.1.4"
       source  = "github.com/YaleUniversity/goss"
     }
   }
-- 
2.46.1

